core/services/ocrcommon/telemetry.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/synchronization/telem"
 	"github.com/smartcontractkit/chainlink/v2/core/utils"
 
-	relaymercury "github.com/smartcontractkit/chainlink-relay/pkg/reportingplugins/mercury"
+	relaymercuryv1 "github.com/smartcontractkit/chainlink-relay/pkg/reportingplugins/mercury/v1"
```

Comment:
Fixing build for now (stick to v1 only) and dealing with telemetry later

Reply:
Maybe this could be `relaymercuryv0`?